site/specs/voice.yml

```diff
@@ -0,0 +1,5173 @@
+openapi: 3.0.3
+info:
+  title: Voice
+  version: 4.1.0
+  contact:
+    name: Bandwidth
+    url: https://support.bandwidth.com
+    email: support@bandwidth.com
+  description: |-
+    The API Specification for Bandwidth's Programmable Voice Platform
+
+    ## Base URL
+
+    `https://voice.bandwidth.com`
+servers:
+  - url: https://voice.bandwidth.com/api/v2
+    description: Production
+paths:
+  /accounts/{accountId}/calls:
+    post:
+      tags:
+        - Calls
+      summary: Create Call
+      description: >-
+        Creates an outbound phone call.
+
+
+        All calls are initially queued. Your outbound calls will initiated at a
+        specific dequeueing rate, enabling your application to "fire and forget"
+        when creating calls. Queued calls may not be modified until they are
+        dequeued and placed, but may be removed from your queue on demand.
+
+
+        <b>Please note:</b> Calls submitted to your queue will be placed
+        approximately in order, but exact ordering is not guaranteed.
+      operationId: createCall
+      parameters:
+        - $ref: '#/components/parameters/accountId'
+      requestBody:
+        $ref: '#/components/requestBodies/createCallRequest'
+      responses:
+        '201':
+          $ref: '#/components/responses/createCallResponse'
+        '400':
+          $ref: '#/components/responses/badRequestError'
+        '401':
+          $ref: '#/components/responses/unauthorizedError'
+        '403':
+          $ref: '#/components/responses/forbiddenError'
+        '404':
+          $ref: '#/components/responses/notFoundError'
+        '405':
+          $ref: '#/components/responses/notAllowedError'
+        '415':
+          $ref: '#/components/responses/unsupportedMediaTypeError'
+        '429':
+          $ref: '#/components/responses/tooManyRequestsError'
+        '500':
+          $ref: '#/components/responses/internalServerError'
+      x-codeSamples:
+        - lang: cURL
+          source: |
+            curl 'https://voice.bandwidth.com/api/v2/accounts/12345/calls' \
+                -X POST \
```

Comment:
There might be som e history here-:) Why are we providing sample code within the spec?

Reply:
The sample code gets automatically injected into the spec file and rendered in our documentation 

you can see an example of that here: 
https://dev.bandwidth.com/apis/voice/#tag/Calls/operation/createCall

![Screen Shot 2022-06-21 at 4 44 36 PM](https://user-images.githubusercontent.com/53190766/174894112-5981e931-cdb2-4078-91cb-7486340cfa75.png)

the code samples are all contained in this repository: 
https://github.com/Bandwidth/api-docs-code-snippets